frontend/src/pages/auth/components/sign-in-form/styles.module.scss

```diff
@@ -0,0 +1,72 @@
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 327px;
+  margin: auto;
+}
+
+.title {
+  align-self: flex-start;
+  margin: 0 0 44px;
+  color: var(--light-green);
+  font-size: 24px;
+  font-family: "Libre Baskerville", sans-serif;
+}
+
+.form {
+  width: 100%;
+  margin-bottom: 18px;
+}
+
+.submitButton {
+  width: 100%;
+  height: 45px;
+  color: var(--white);
+  background: var(--dark-green);
+  border-radius: 7px;
+}
+
+.passwordToggle {
+  display: contents;
+}
+
+.fieldset {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+  margin: 0 0 18px;
+  padding: 0;
+  border: none;
+}
+
+.passwordInputWrapper {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+}
+
+.icon {
+  position: absolute;
+  top: 35px;
+  right: 20px;
+  width: 16px;
+  height: 16px;
+  cursor: pointer;
+  filter: invert(61%) sepia(7%) saturate(75%) hue-rotate(201deg) brightness(93%)
+    contrast(89%);
```

Comment:
<img width="782" alt="image" src="https://github.com/BinaryStudioAcademy/bsa-2023-writorium/assets/19575839/edf4bd9e-1418-476a-abc0-e8fd2ded68ca">

Not sure if it would be stable